melodic/distribution.yaml

```diff
@@ -2069,6 +2069,21 @@ repositories:
       url: https://github.com/OTL/cv_camera.git
       version: master
     status: maintained
+  cyglidar_d1:
+    doc:
+      type: git
+      url: https://github.com/CygLiDAR-ROS/cyglidar_d1.git
+      version: v0.1.1
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/CygLiDAR-ROS/cyglidar_d1.git
+      version: v0.1.1
```

Comment:
For how to release, check http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease.
We can add this as a source package for the moment:

```suggestion
```